test/utils/utilities.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @param {Array} pageData - data returned from update request
+ * @param {Array} tuples - tuples sent to the database for update
+ * @param {string} sortBy - name of column to compare the pageData/tuples by so they are similarly sorted
+ * verifies that the returned data is the same as the submitted data
+ */
+exports.checkPageValues = function(pageData, tuples, sortBy) {
+    pageData.sort(function(a, b) {
+        return a[sortBy] - b[sortBy];
+    });
+
+    tuples.sort(function(a, b) {
+        return a._data[sortBy] - b._data[sortBy];
```

Comment:
This is only ever using the `sortBy` value (which I use an int column always to do it). But I will change it to `localeCompare` for robustness.